src/main/java/com/onlinebookstore/model/Book.java

```diff
@@ -32,6 +38,13 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
+
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
do not use EAGER
use @EntityGraph in repository

Reply:
Ok, done